### PR TITLE
Titov task2

### PR DESCRIPTION
**Pull request by titov-evgeni**

### CODE REVIEW

**Review by MikitaTsiarentsyeu — CHANGES_REQUESTED** [review]
image

a minor issue but still, it's here

**Comment by titov-evgeni** [comment]
i made changes

File: Tasks/Task2/task_2.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+текстовый вывод времени, введённого с консоли.
+Для получения текущего времени использовать модуль datetime.
+"""
+
+from datetime import datetime
+
+
+def time_string(time):
+    hours_dict = {
+        0: "Полночь",
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Task2/task_2.py:14]: nice touch

**Review by MikitaTsiarentsyeu — APPROVED** [review]
Good job, the logic is totally correct, I have no nitpicks. Your score is 9.0.

**Pull request merged**